### ISSUE TITLE
New Cog: Distance Between Locations 

### ISSUE THREAD

**Issue by Exper1mental** [ClemBotProject/ClemBot#630]
The general idea is to have a command where you input two locations, and the bot will report back the distance between them. The geocode.xyz API I use for the weather command could likely be reused for this purpose, at least in terms of converting user input into a more computer-readable format (if needed).

I'm mainly thinking of straight-line distance but it could also be cool to add a way to see what the distance would be by road, bike, train, etc. This could then be expanded further into providing estimated travel time and even a link to navigation on Apple Maps, Google Maps, DuckDuckGo, etc.

Examples:
`!distance "New York, NY, USA" to "Clemson, SC, USA"`
`!dist road "Greenville, SC, USA" to "Central, SC, USA"`
`!traveltime road "Chicago, IL, USA" to "Toronto, ON, Canada"`

**Comment by chasepg0821** [comment]
@jkriste Could I be assigned to this issue?

**Comment by chasepg0821** [comment]
I'd be happy to implement it into SockBot. I'll open an issue there. Would the riddles cog be a similar situation too? because I'd be happy to implement that one here or in SockBot too. Thanks!

**Comment by jkriste** [comment]
@chasepg0821 Yes, the riddles cog would be a similar situation as well. You can create an issue for that one too and I'll assign them to you when I can.

I'll go ahead and close this issue. Thank you!
